src/material_pool.hpp

```diff
@@ -94,6 +94,11 @@ namespace wr
 			m_material_data.m_constant_data[uint32_t(type) >> 16] = x;
 		}
 
+		template<MaterialConstant type>
+		void SetConstant(typename std::enable_if<uint8_t(type) == 0, bool>::type x) {
```

Comment:
casting here to uint8 is isconsistent.

Reply:
also wtf is this setconstant code. this only works for 2 types right now?

Reply:
looked it up and I think you emilio misunderstands how the system works.

Reply:
![image](https://user-images.githubusercontent.com/5305297/59716295-07d75880-9216-11e9-97b9-e35776af3ba1.png)

Reply:
oh wtf that worked? fucking sick github.

Reply:
Old debug code as well, and yeah misunderstood the system. Removed as it was not needed anyway.